### DISCUSSION TITLE
PocketHost

### DISCUSSION THREAD

**Discussion by RobbeDeClerck** [pockethost/pockethost#234] [Q&A]
Is [PocketHost](https://pockethost.io) down? I've been working on it for a bit and noticed while working everything went down..

**Comment by NatumanyaGuy** [comment]
> Is [PocketHost](https://pockethost.io) down? I've been working on it for a bit and noticed while working everything went down..

Seems so, I kept getting ` Error: certificate has expired` in my app; and when I visited the [site](https://pockethost.io/), saw the, `"Your connection is not private!"`?

**Comment by benallfree** [comment]
Moving to #223
